src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -266,6 +266,8 @@ def load_arguments(self, _):
                    help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
         c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
                    help="Geographic location where Function App will be hosted. Use 'functionapp list-consumption-locations' to view available locations.")
+        c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
```

Comment:
This flag will only be used with consumption plans, right? If yes, I would suggest the help text would be `host function app on linux consumption plan`, or even name the arg like `--use-linux-consumption-plan`. You can feel free to change to any other text, as long as we make this point clear; otherwise it could become confusing as it is well known that apps are to infer the linux worker from its ASP.
Also we should emit out an error if the `--plan` is also provided